src/org/yccheok/jstock/gui/charting/ChartJDialog.java

```diff
@@ -78,7 +85,7 @@
  *
  * @author  yccheok
  */
-public class ChartJDialog extends javax.swing.JDialog {
+public class ChartJDialog extends javax.swing.JFrame implements WindowListener {
```

Comment:
I can't really recall why I choose to use JDialog instead of JFrame last time.

I can see the benefit of having JFrame - Able to open multiple charts and able to minimize some of them to task bar, to avoid from blocking each other visibility.

The only short coming I can think of, is task bar will become more clumsy. (Please refer to attachment)

If we choose JFrame over JDialog, will we miss any benefit provided by JDialog?

![2015-04-03 02_10_53-](https://cloud.githubusercontent.com/assets/308276/6970242/c32cb486-d9a6-11e4-9fda-553208ce3f18.png)
![2015-04-03 02_11_06-](https://cloud.githubusercontent.com/assets/308276/6970245/c5c364c4-d9a6-11e4-88d2-b0808bb38c0b.png)